### PR TITLE
fully externalize exporter; untested

### PR DESCRIPTION
**Pull request by jywarren**
Integrating work from https://github.com/publiclab/mapknitter-exporter/issues/5 to get all Ruby/Gdal/ImageMagick exporting code out of the core app!

### CODE REVIEW

**Comment by jywarren** [comment]
Hmm, the second failure is: `assert File.exist?('public/warps/saugus-landfill-incinerator/1-geo.tif')`

I wonder if this has to do with the change in `slug` and `id`?

File: app/models/warpable.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require "mapknitterExporter"
```
Comment (jywarren) [inline app/models/warpable.rb]: Hmm, @icarito i'm seeing `File does not exist: mapknitterExporter` from this. I copied the Sinatra app config... any ideas?

**Comment by jywarren** [comment]
Hi @sashadev-sky -- note that this might touch your work on tracking external status... just FYI!

**Comment by sashadev-sky** [comment]
@jywarren noted!

**Comment by sashadev-sky** [comment]
> Hmm, the second failure is: `assert File.exist?('public/warps/saugus-landfill-incinerator/1-geo.tif')`
> 
> I wonder if this has to do with the change in `slug` and `id`?

could this be carried over from the potential problem we were discussing in this PR? https://github.com/publiclab/mapknitter/pull/341

**Comment by gauravano** [comment]
hey @jywarren @sashadev-sky, anything you want to update on this PR?

**Comment by sashadev-sky** [comment]
@gauravano good catch! Not sure, a @jywarren will have to advise!

**Comment by jywarren** [comment]
So, I think actually we'll be going directly to the Cloud exporter in https://github.com/publiclab/mapknitter/pull/693, so we'll get that working fully with a UI, then start stripping out the exporter code from this project entirely! cc @ViditChitkara @divyabaid16

**Pull request closed without merging**